aio/aio-proxy/aio_proxy/request/search_params_builder.py

```diff
@@ -1,229 +1,52 @@
-                str_to_list(clean_parameter(request, param="include_admin")),
-                admin=True,
-            ),
-        )
-        SearchParamsBuilder.check_and_validate_params(request, params)
-        return params
+    PARAMETER_MAPPING = {
+        "q": "terms",
+        "limite_matching_etablissements": "matching_size",
+        "nature_juridique": "nature_juridique_unite_legale",
+        "date_naissance_personne_min": "min_date_naiss_personne",
+        "date_naissance_personne_max": "max_date_naiss_personne",
+        "etat_administratif": "etat_administratif_unite_legale",
+        "activite_principale": "activite_principale_unite_legale",
+        "code_commune": "commune",
+        "tranche_effectif_salarie": "tranche_effectif_salarie_unite_legale",
+        "est_ess": "economie_sociale_solidaire_unite_legale",
+        "long": "lon",
+    }
 
     @staticmethod
-    def get_geo_search_params(request):
-        params = SearchParams(
-            page=parse_and_validate_page(request),
-            per_page=parse_and_validate_per_page(request),
-            lat=parse_and_validate_latitude(request),
-            lon=parse_and_validate_longitude(request),
-            radius=parse_and_validate_radius(request),
-            activite_principale_unite_legale=validate_activite_principale(
-                str_to_list(clean_parameter(request, param="activite_principale"))
-            ),
-            section_activite_principale=validate_section_activite_principale(
-                str_to_list(
-                    clean_parameter(request, param="section_activite_principale")
-                )
-            ),
-            matching_size=parse_and_validate_matching_size(request),
-            minimal=parse_and_validate_bool_field(request, param="minimal"),
-            include=validate_selected_fields(
-                str_to_list(clean_parameter(request, param="include"))
-            ),
-            include_admin=validate_selected_fields(
-                str_to_list(clean_parameter(request, param="include_admin")),
-                admin=True,
-            ),
-        )
-        validate_results_window(params.page, params.per_page)
-        return params
+    def map_request_parameters(request):
+        # Extract all query parameters from the request
+        request_params = request.rel_url.query
+
+        mapped_params = {}
+        # Include parameters specified in PARAMETER_MAPPING with mapping
+        for http_param, model_field in SearchParamsBuilder.PARAMETER_MAPPING.items():
+            param_value = request_params.get(http_param, None)
+            if param_value is not None:
+                mapped_params[model_field] = param_value
+        # Include parameters not specified in PARAMETER_MAPPING without mapping
+        for key, value in request_params.items():
+            if key not in SearchParamsBuilder.PARAMETER_MAPPING:
+                mapped_params[key] = value
+        return mapped_params
```

Comment:
For param in key, value in request_params.items():
	shouldBeRemap = key in SearchParamsBuilder.PARAMETER_MAPPING

	if shouldBeRemap
		remap
	else 
		default